website/docs/cartesian/cartesian-chart.md

```diff
@@ -102,6 +102,7 @@ The `axisOptions` is an optional prop allows you to configure the axes and grid
 | :-----------------: | ---------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 |     **`font`**      | <pre>SkFont &#124; null</pre>                                                                        | Defines the font to be used for axis labels. If not provided, then no labels will be rendered. This font object is typically returned from Skia’s `useFont` hook.                                                                                                                                                                  |
 |   **`tickCount`**   | <pre>number &#124; {<br /> x: number; <br /> y: number;<br />}</pre>                                 | Defines the number of ticks to be rendered on both axes, or if provided an object, specific to x and y. If not provided, then the chart will attempt to choose a reasonable number of ticks based on the size of the chart. <br /><br />Note: This is an approximation; the scale may return more or fewer values depending on the domain, padding, and axis labels.                                                                                                          |
+|   **`tickValues`**   | <pre>[number] &#124; {<br /> x: [number]; <br /> y: [number];<br />}</pre>                                 | Defines the explicit set of numeric tick values to be rendered on both axes, or if provided an object, specific to x and y. The tickValues prop is used to specify the values of each tick, so we only accept numeric values. Use the `formatXLabel` or `formatYLabel` options to customize how the ticks should be labeled. <br /><br />Note: If `tickCount` value is also provided, it will be used to downsample the provided `tickValues` array to the specified length.                                                                                                          |
```

Comment:
It might be worth also adding a note about how it seems like the domain values take precedence over the `tickValues` if it's present?

Reply:
@zibs technically, the domain should never take precedence over the tickValues, per se.

Instead, if a domain is NOT present, we attempt to create our own domain using the min/max of any tickValues provided (and if not, we fall back to getting the domain from the min/max of the data itself). In the end, we still use all of the tickValues to place the ticks on the chart.

This can create some fun cases where the provided `tickValues` may land outside of the specified domain which may benefit from a note. But, I wasn't sure how best to document this, so I chose not to include it (as is the case with the [Victory docs on tickValues](https://commerce.nearform.com/open-source/victory/docs/victory-axis#tickvalues))

Reply:
Oh I see, okay great!